src/utils/createContextWrapper.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+/**
+ * Creates new trigger class that injects context into overlay.
+ */
+export default function createContextWrapper(Trigger, propName) {
+  return function (contextTypes) {
+    class ContextWrapper extends React.Component {
```

Comment:
Added in https://github.com/taion/react-bootstrap/commit/6f2e03646207c1e31a24ecf68866d69e33d0add4